tests-hivemq-operator/build.gradle.kts

```diff
@@ -19,24 +19,31 @@ repositories {
 /* ******************** test ******************** */
 
 dependencies {
-    testImplementation("org.assertj:assertj-core:${property("assertj.version")}")
-    testImplementation("org.codehaus.groovy:groovy-all:${property("groovy.version")}")
-    testImplementation("org.junit.jupiter:junit-jupiter:${property("junit.version")}")
-    testImplementation("org.junit.jupiter:junit-jupiter-api:${property("junit.version")}")
-    testImplementation("org.testcontainers:testcontainers:${property("testcontainers.version")}")
-    testImplementation("org.testcontainers:k3s:${property("testcontainers.version")}")
-    testImplementation("org.testcontainers:junit-jupiter:${property("testcontainers.version")}")
-    testImplementation("org.slf4j:slf4j-api:${property("slf4j.version")}")
-    testImplementation("org.slf4j:slf4j-simple:${property("slf4j.version")}")
-    testImplementation("com.hivemq:hivemq-mqtt-client:${property("hivemq-client.version")}")
-    testImplementation("io.fabric8:kubernetes-client:${property("fabric8.version")}")
-    testImplementation("org.bouncycastle:bcprov-jdk15on:${property("bouncycastle.version")}")
-    testImplementation("org.bouncycastle:bcpkix-jdk15on:${property("bouncycastle.version")}")
-    testImplementation("org.awaitility:awaitility:${property("awaitility.version")}")
-    testImplementation("ch.qos.logback:logback-classic:${property("logback.version")}")
-    testImplementation("ch.qos.logback:logback-core:${property("logback.version")}")
-    testImplementation("org.slf4j:slf4j-api:${property("slf4j.version")}")
-    testImplementation("org.testcontainers:hivemq:${property("testcontainers.version")}")
+    // Testing
+    testImplementation(libs.assertj)
+    testImplementation(libs.awaitility)
+
+    // JUnit
+    testImplementation(libs.junit.jupiter)
+    testImplementation(libs.junit.jupiter.api)
```

Comment:
I'm pretty sure we don't need the API dependency if we have `junit-jupiter` as dependency. That should be a transitive dependency.

Reply:
good one, thanks